modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/RosettaError.scala

```diff
@@ -42,6 +42,13 @@ object RosettaError extends IntEnum[RosettaError] with RosettaErrorEncoder {
         retriable = false,
         description = Some("Unable to decode the transaction.")
       )
+  case object UnsupportedOperation
+      extends RosettaError(
+        value = 7,
+        message = "Unsupported operation",
+        retriable = false,
+        description = Some("The operation is not supported.")
+      )
```

Comment:
according  to previous comment, this would never happen so most likely it's unused/unnecessary

Reply:
This is no used in the route when there are more than 1 signature.